blabla/sentence_processor/syntactic_feature_engine.py

```diff
@@ -43,6 +43,21 @@ def num_prepositional_phrases(const_pt):
     return len(pp_chunks)
 
 
+def num_t_units(const_pt):
+    """Returns the number of T-Units
```

Comment:
Can we consistently capitalize as "T-unit" everywhere? I.e. uppercase T, lowercase u.